web/react/utils/utils.jsx

```diff
@@ -911,6 +911,18 @@ export function isBrowserEdge() {
     return window.naviagtor && navigator.userAgent && navigator.userAgent.toLowerCase().indexOf('edge') > -1;
 }
 
+export function getDirectChannelName(id, otherId) {
```

Comment:
I found functions to do the opposite of this (get the ids/users from the direct channel name), but not create direct channel names. The only place I found this being done is the sidebar which is where I took this code from.